protocol/app/upgrades_test.go

```diff
@@ -0,0 +1,80 @@
+package app_test
+
+import (
+	"errors"
+	"testing"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/module"
+	"github.com/cosmos/cosmos-sdk/x/upgrade/types"
+	"github.com/dydxprotocol/v4-chain/protocol/app"
+	"github.com/dydxprotocol/v4-chain/protocol/app/upgrades"
+	testapp "github.com/dydxprotocol/v4-chain/protocol/testutil/app"
+	"github.com/stretchr/testify/require"
+)
+
```

Comment:
Meta:
1. should we add tests for `setupUpgradeStoreLoaders`?
2. should we add tests for `scheduleForkUpgrade`? i.e. no duplicate heights, fork upgrade gets triggered on the height

Reply:
Both `setupUpgradeStoreLoaders` and `scheduleForkUpgrade` make more sense as integration tests since we will get little value out of the unit test.

Reply:
reopened the mentioned ticket + added comments!